modulefiles/hera.intel/fv3

```diff
@@ -21,7 +21,6 @@ module load sutils
 ##
 module load intel/18.0.5.274
 module load impi/2018.0.4
-module load netcdf/4.7.0
```

Comment:
Do we not need netcdf lib? I am curious why module files on other platforms are removed e.g. wcoss, cheyenne etc?